lib/YaST/Bootloader/BootcodeOptionsPage.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Bootloader
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Bootloader::BootcodeOptionsPage;
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my ($self) = @_;
+    $self->{cmb_bootloader}        = $self->{app}->combobox({id => '"Bootloader::LoaderTypeWidget"'});
+    $self->{cb_write_to_partition} = $self->{app}->checkbox({id => 'boot'});
+    $self->{cb_bootdev}            = $self->{app}->checkbox({id => 'mbr'});
+    $self->{cb_custom_boot}        = $self->{app}->checkbox({id => 'custom'});
+    $self->{cmb_mbr_flag}          = $self->{app}->combobox({id => '"Bootloader::PMBRWidget"'});
+    $self->{cb_trusted_boot}       = $self->{app}->checkbox({id => '"Bootloader::TrustedBootWidget"'});
+    $self->{cb_generic_to_mbr}     = $self->{app}->checkbox({id => '"Bootloader::GenericMBRWidget"'});
+    $self->{cb_set_active_flag}    = $self->{app}->checkbox({id => '"Bootloader::ActivateWidget"'});
+    $self->{tb_boot_options}       = $self->{app}->tab({id => '"CWM::DumbTabPager"'});
+    $self->{btn_next}              = $self->{app}->button({id => 'next'});
+    return $self;
+}
+
+sub get_boot_options {
+    my ($self) = @_;
+    my $bool = $self->{tb_boot_options}->selected_tab();
+    return "$bool";
+}
+
+sub get_bootloader_type {
+    my ($self) = @_;
+    return $self->{cmb_bootloader}->value();
+}
+
+sub get_write_to_mbr {
+    my ($self) = @_;
+    my $bool = $self->{cb_bootdev}->is_checked();
```

Comment:
please apply the same to the rest of the unnecessary `$bool` variables.

Reply:
They are needed, as we need quotes around the returned value to avoid this https://chat.suse.de/channel/qe-yast?msg=tssoN8EHXW6Zg9oPL

Reply:
I was discussing with Alex and we should avoid quotation in both sides, so not be afraid of this errors with Boolean objects. Stringify in both sides is hacky. We might find better ways to display the error in case is not clear when is showed as comparation of boolean. We have to research that in another task: https://metacpan.org/pod/YAML::PP#boolean

Reply:
but I'm just talking about a better visualization of the error, that could be researched it if needed.
For now Alex and me think that we should remove quotation from test data and avoid this stringifying for each value you collect in the controller, please remove this in both sides.

Reply:
well ok, but then we have the problem shown above.